NuGet.Config

```diff
@@ -4,4 +4,7 @@
     <add key="NuGet.org" value="https://api.nuget.org/v3/index.json" />
     <add key="xunit" value="https://www.myget.org/F/xunit/api/v3/index.json" />
   </packageSources>
+  <disabledPackageSources>
+    <add key="xunit" value="true" />
+  </disabledPackageSources>
```

Comment:
Just remove the `xunit` package source completely. It's available on nuget.org now.